lib/puma/dsl.rb

```diff
@@ -408,22 +408,26 @@ def threads(min, max)
     #     key: path_to_key,
     #     ssl_cipher_filter: cipher_filter, # optional
     #     verify_mode: verify_mode,         # default 'none'
+    #     verification_flags: flags,        # optional, not supported by JRuby
     #     keystore: path_to_keystore,
     #     keystore_pass: password
     #   }
     def ssl_bind(host, port, opts)
       verify = opts.fetch(:verify_mode, 'none').to_s
       no_tlsv1 = opts.fetch(:no_tlsv1, 'false')
       no_tlsv1_1 = opts.fetch(:no_tlsv1_1, 'false')
-      ca_additions = "&ca=#{opts[:ca]}" if ['peer', 'force_peer'].include?(verify)
+
+      additions = +''
```

Comment:
Given that Puma is currently supporting Ruby 2.2:

```suggestion
      additions = ''.dup
```